gen/cow/cow.hoon

```diff
@@ -0,0 +1,59 @@
+::  The classic unix "cowsay" implemented in hoon
+::
+::  To see a cow say a quote from gen/cow/quotes.txt run:
+::
+::  +cow/cow
+::
+/-  sole
+/+   generators, pretty-file
+=,  generators
+:-  %ask
+|=  [[now=@da eny=@uvJ bec=beak] ~ ~]
+=/  max-length  80
+=/  file  (pretty-file .^(noun (cat 3 %c %x) `path`/(scot %p p.bec)/[q.bec]/(scot r.bec)/gen/cow/quotes/txt))
+=/  line  (snag (~(rad og eny) (lent file)) file)
+=/  tongue  "U"
+=/  thoughts  "o"
+=/  eyes  "oo"
+=<  (cow)
+|%
+++  cow
+    |=  ~
+    ^-  (sole-request:sole (cask tang))
+    =/  bubble  (turn (message) bars)
+    =/  horizontal-border  leaf+(weld "  " (reap (sub (lent (snag 0 bubble)) 5) '-'))
+    =/  slant-gap  (reap (sub (lent (snag 0 bubble)) 5) ' ')
+    =/  top-slant  leaf+" /{slant-gap}\\"
+    =/  bottom-slant  leaf+" \\{slant-gap}/"
+    %+  produce  %tang
+    %+  weld
+      :~  leaf+"            ||     ||"
+          leaf+"          {tongue} ||----w |"
+          leaf+"        (__)\\       )\\/\\"
+          leaf+"      {thoughts} ({eyes})\\_______"
+          leaf+"     {thoughts}  ^__^"
+      ==
+      (weld (weld ~[horizontal-border bottom-slant] bubble) ~[top-slant horizontal-border])
```

Comment:
Indenting on this is a bit off. Also, if you want to `weld` more than two things, you can use the `;:` rune. 

```hoon
;:  weld
  ~[horizontal-border bottom-slant]
  bubble
  ~[top-slant horizontal-border]
==
``

For shorter inputs, the short form might come in handy: `:(weld one two three)`.